File: Assets/Scripts/Core/Rules/RuleSettings.cs
using System;

namespace Parkiller.Core
{
    [Serializable]
    public class RuleSettings
    {
        public int entryRoll = 6;
        public bool grantExtraTurnOnSix = true;
        public bool thirdConsecutiveSixForfeitsMove = true;
        public bool captureSendsToYard = true;
    }
}

File: src/core/rules/ruleSettings.ts
export interface RuleSettings {
  entryRoll: number
  grantExtraTurnOnSix: boolean
  thirdConsecutiveSixForfeitsMove: boolean
  captureSendsToYard: boolean
}

export function defaultRuleSettings(): RuleSettings {
  return {
    entryRoll: 6,
    grantExtraTurnOnSix: true,
    thirdConsecutiveSixForfeitsMove: true,
    captureSendsToYard: true,
  }
}
